src/sql/workbench/api/node/extHostConnectionManagement.ts

```diff
@@ -21,7 +21,9 @@ export class ExtHostConnectionManagement extends ExtHostConnectionManagementShap
 	public $getCurrentConnection(): Thenable<azdata.connection.ConnectionProfile> {
 		let connection: any = this._proxy.$getCurrentConnection();
 		connection.then((conn) => {
-			conn.providerId = conn.providerName;
+			if (conn) {
```

Comment:
Added this because I saw a bunch of silent errors when working with different extension context menus.

Reply:
makes sense..may even make sense to add `&& conn.providerName` to the condition to avoid null'ing providerId, but can't be certain without further inspection.